src/common/models/Projection.ts

```diff
@@ -161,6 +173,13 @@ export class Projection {
       summaryWithTimeseries.actuals.cumulativeDeaths;
     this.currentCumulativeCases =
       summaryWithTimeseries.actuals.cumulativeConfirmedCases;
+    this.currentContactTracerMetric = this.contractTracers
+      .filter(x => x !== null)
```

Comment:
Just confirming you won't hit `undefined` here, in which case you could do `!=` instead